app/views/users/following.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <div class="col md-6 offset-3">
+    <h1>
+      <%= @user.username %>
+    </h1>
+
+    <ul class="nav nav-pills nav-justified">
+      <li class="nav-item">
+        <%= link_to "Posts", user_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Liked photos", liked_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+          <%= link_to "Feed", feed_path(@user.username), class: "nav-link" %>
+      </li>
+      
+      <li class="nav-item">
+        <%= link_to "Followers", followers_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link" %>
+      </li>
+
+
+    </ul>
+  </div>
+</div>
+
+<% @user.leaders.each do |leader| %>
+  <%= render "follow_requests/following", leader: leader %>
```

Comment:
You are off to a good start; however, this logic is not quite correct. Your lists are showing up multiple times. Try removing the partial view template and adding the logic for your following directly to this view template.